mongo/cursor_cache_test.go

```diff
@@ -20,6 +20,10 @@ func (m *mockServer) MinRTT() time.Duration {
 	return time.Duration(0)
 }
 
+func (m *mockServer) RTT90() time.Duration {
+	return time.Duration(0)
+}
```

Comment:
In the new version of the driver the `Server` interface added this `RTT90` function that needed to be added to appease the type system